page.php

```diff
@@ -63,11 +64,15 @@
 Context::set_campaign_datalayer( $context, $page_meta_data );
 Context::set_utm_params( $context, $post );
 
+$page_category            = $data_layer['page_category'];
+$listing_pages_categories = [ 'Tag Page' ];
+
 $context['post']                = $post;
 $context['social_accounts']     = $post->get_social_accounts( $context['footer_social_menu'] );
-$context['page_category']       = $data_layer['page_category'];
+$context['page_category']       = $page_category;
 $context['post_tags']           = implode( ', ', $post->tags() );
 $context['custom_body_classes'] = 'brown-bg ';
+$context['hide_background']     = HideListingPagesBackground::is_active() && in_array( $page_category, $listing_pages_categories, true );
```

Comment:
I've had to add this here as well as in `tag.php` in case a tag has a redirect page.

Reply:
However I'm not sure it's the way to go, since redirect pages could be anything and therefore not even really listing pages, maybe in that case they should keep their background? 🤔

Reply:
Indeed after discussing with the design team, we decided to keep the background in redirect pages for now so code has been updated!